### PR TITLE
yaml2ck man page

### LINKED ISSUE #1361: man page missing for `yaml2ck`
**Problem description**

While `ck2yaml`, `cti2yaml` and `ctml2yaml` have man pages, it is missing for `yaml2ck` (new in Cantera 3.0; introduced with #1286).

**Steps to reproduce**

```
$ man yaml2ck
No manual entry for yaml2ck
```

**System information**

- Cantera version: 3.0.0a2
- OS: any
- Python/MATLAB/other software versions: N/A

### PR DESCRIPTION
**Pull request by mefuller**
**Changes proposed in this pull request**

Add man page for `yaml2ck` (closes #1361)

**If applicable, provide an example illustrating new features this pull request is introducing**

**Checklist**

- [x] The pull request includes a clear description of this code change
- [x] Commit messages have short titles and reference relevant issues
- [x] Build passes (`scons build` & `scons test`) and unit tests address code coverage
- [x] Style & formatting of contributed code follows [contributing guidelines](https://github.com/Cantera/cantera/blob/main/CONTRIBUTING.md)
- [x] The pull request is ready for review

### CODE REVIEW

File: platform/posix/man/yaml2ck.1 [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+of the elements and species, thermodynamic data, as well as the reactions
+and transport data (if any).
+.TP
+.B --phase-name=\fIPHASE\fR
+Name of the phase to load from the input. If unspecified, the first phase
+listed in the input file will be loaded.
+.TP
+.B --mechanism=\fIMECH\fR
+The path to the output mechanism file. Optional. If not provided, the name
+of the input file will be used, with the extension replaced by '.ck'.If
```
Comment (ischoegl) [inline platform/posix/man/yaml2ck.1]:
```suggestion
of the input file will be used, with the extension replaced by '.ck'. If
```
Curiously, `yaml2ck.py` has the same formatting glitch.

File: platform/posix/man/yaml2ck.1 [hunk 2]
```diff
@@ -0,0 +1,58 @@
+.TH "yaml2ck" 1 "12 Aug 2022" "yaml2ck" \" -*- nroff -*-
+.ad l
+.nh
+.SH NAME
+yaml2ck \- convert Cantera YAML input files to Chemkin-II-format mechanisms
```
Comment (ischoegl) [inline platform/posix/man/yaml2ck.1]:
```suggestion
yaml2ck \- convert Cantera YAML input files to Chemkin-format mechanisms
```

`man` text should be equivalent to `yaml2ck --help`. I don't think that the `-II` suffix matters?

**Review by ischoegl — CHANGES_REQUESTED** [review]
Thanks for creating the missing `man` file! I just noticed two minor things.

File: platform/posix/man/yaml2ck.1 [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+.SH NAME
+yaml2ck \- convert Cantera YAML input files to Chemkin-format mechanisms
+
+.SH SYNOPSIS
+.B yaml2ck [\fIOPTIONS\fR] \fIINPUT\fR
+
+.SH DESCRIPTION
+
+.I yaml2ck
+converts a Cantera YAML input file to Chemkin-II-format reaction mechanism.
```
Comment (ischoegl) [inline platform/posix/man/yaml2ck.1]:
```suggestion
converts a Cantera YAML input file to Chemkin-format reaction mechanism.
```
see other comment :laughing:

File: platform/posix/man/yaml2ck (zoe's conflicted copy 2022-08-13).1 [hunk 4]
```diff
@@ -0,0 +1,58 @@
+.TH "yaml2ck" 1 "12 Aug 2022" "yaml2ck" \" -*- nroff -*-
```
Comment (ischoegl) [inline platform/posix/man/yaml2ck (zoe's conflicted copy 2022-08-13).1]: Not sure where this file came from. Once it is removed, this should be good to go!

Comment (mefuller) [reply platform/posix/man/yaml2ck (zoe's conflicted copy 2022-08-13).1]: oops - looks like Dropbox strikes again

**Review by ischoegl — APPROVED** [review]

**Review by ischoegl — APPROVED** [review]
Thanks, @mefuller - this looks great ... I will merge as soon as the CI finishes.

**Pull request merged**